app/components/list/filters/filters.tsx

```diff
@@ -68,7 +61,17 @@ export const Filters = ({
         className
       )}
     >
-      <Form ref={formRef} className="w-full">
+      <Form
+        ref={formRef}
+        className="w-full"
+        onSubmit={(e) => {
+          const formData = new FormData(e.currentTarget);
+          const s = formData.get("s") as string;
+          if (s) {
+            setSearchParams((prev) => ({ ...prev, s }));
```

Comment:
Okey so here I am not sure about this. So a few things:
1. the form submits and then we also change the params? Wouldnt that do a double submit
2. Shouldnt we merge all existing params? what about stuff like `page` or `per_page`?